resources/moneropedia/mnemonicseed.md

```diff
@@ -2,7 +2,7 @@
 layout: moneropedia
 entry: "Mnemonic Seed"
 terms: ["mnemonic-seed", "mnemonic"]
-summary: "a 13 or 25 word phrase used to backup a Monero account, available in a number of languages"
+summary: "a 13 or 25 word phrase used to back up a Monero account, available in a number of languages"
```

Comment:
s/back up/backup

`back up` is to reverse (like a truck/lorry moving backwards) `backup` or `back-up` is to save in other places for redundancy.

Reply:
Fixed.